src/test/resources/Logout.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Login
+
+  An a user
+  I want to sign in
+  So I can use the app
+
+  @automated
+  Scenario: User can sign in with valid value
```

Comment:
`Scenario: User can sign in with valid value` sửa thành  ` Scenario: User can sign out the app`